cmd/kops/integration_test.go

```diff
@@ -67,39 +67,10 @@ func runTest(t *testing.T, clusterName string, srcDir string) {
 	factoryOptions := &util.FactoryOptions{}
```

Comment:
So is this actually an integration test that will build a cluster? We have unit, integration, e2e. What do the names mean?